### PR TITLE
Fix a typo and add a few hyperlinks

### PR DESCRIPTION
**Pull request by fatosmorina**
Fix a typo and add a few hyperlinks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ A more detailed report can be found [here](https://github.com/eluzhnica/ISFA/blo
 The ISFA is currently composed of:
 
 - A parser for math formulas found in the OEIS.
-- A program that find relations between OEIS sequences. Relations in simple text format (due to space limitations) are available [here](https://kwarc.info/datahost/).
-- A MathWebSearch instance for OEIS documents (http://oeissearch.mathweb.org) (thanks to Mihnea Iancu)
+- A program that finds relations between OEIS sequences. Relations in simple text format (due to space limitations) are available [here](https://kwarc.info/datahost/).
+- A MathWebSearch instance for [OEIS documents](http://oeissearch.mathweb.org) (thanks to Mihnea Iancu)
```
Comment (eluzhnica) [inline README.md]: Ideally, the hyperlink should be on 'MathWebSearch instance`.

**Review by eluzhnica — CHANGES_REQUESTED** [review]
Thanks for the PR Fatos! 
Left one comment there.

**Comment by fatosmorina** [comment]
@eluzhnica My pleasure. 

A few new changes have been committed.

**Review by eluzhnica — APPROVED** [review]
All good. Thank you!

**Comment by fatosmorina** [comment]
You are welcome.

**Pull request merged**